datafusion/physical-expr/src/window/aggregate.rs

```diff
@@ -59,6 +59,11 @@ impl AggregateWindowExpr {
             window_frame,
         }
     }
+
+    /// Get aggregate expr of AggregateWindowExpr
+    pub fn get_aggregate_expr(&self) -> &Arc<dyn AggregateExpr> {
+        &self.aggregate
```

Comment:
`get_aggregate_expr ` and `get_built_in_func_expr ` return diff trait object ,  i think can not extract them in `WindowExpr `